### ISSUE TITLE
[Bug][A11Y][Phone number] Improve vocalization of phone number input

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#497]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

.

### OS Version

Android and iOS

### App version

0.6.0 (509)

### Describe the issue

The whole phone number component (grouped select country + input) is read "Phone number, Phone number" + hint "Include you full number without spaces"

Remove the second "Phone Number" which is linked to the input I guess.

Remove the hint from the group

<img width="422" height="521" alt="" src="https://github.com/user-attachments/assets/1a4b6821-158b-4cb5-badd-21fad824079c" />

### Expected Behavior

When screen reader focus is on the group (select + input), label should be only "Phone Number"
When focus goes on input field, we should hear "Phone Number" + hint "Include you full number without spaces" + double tap to edit

**Comment by B3nz01d** [comment]
@AhmedAmineZr in the version 591, when the screen reader focus is on the group it reads the hint and it should not

**Comment by AhmedAmineZr** [comment]
> [@AhmedAmineZr](https://github.com/AhmedAmineZr) in the version 591, when the screen reader focus is on the group it reads the hit and it should not

this the behavior expected that @pya35 shared with me 
When screen reader focus is on the group (select + input), label should be only "Phone Number"

**Comment by AhmedAmineZr** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (alpha) (592)
Version: 0.7.0
Build type: Alpha
Build details (GitHub): 326
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (Accessibility)

**Comment by AhmedAmineZr** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (592)
Version: 0.7.0
Build type: Alpha
Build details (GitHub): 326
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (Accessibility)

**Comment by pya35** [comment]
clos

**Comment by B3nz01d** [comment]
Finished during sprint # 25
